pkg/cli/quit.go

```diff
@@ -113,7 +113,8 @@ func doDrain(
 	})
 	if errors.HasType(err, (*contextutil.TimeoutError)(nil)) || grpcutil.IsTimeout(err) {
 		log.Infof(ctx, "drain timed out: %v", err)
-		err = errors.New("drain timeout")
+		err = errors.New("drain timeout, consider adjusting --drain-wait, especially under " +
```

Comment:
maybe try this:
```go
err = errors.WithHint(err, "Consider adjusting --drain-wait, especially under " +
"custom server.shutdown.{drain,query,lease_transfer}_wait cluster settings.")
```

(assuming the hint shows up in the command output)

Reply:
It does not, unfortunately:

```
$ ./cockroach node drain --insecure --drain-wait=1ns
node is draining... 
ERROR: operation timed out.

operation "drain" timed out after 1ns (took 0s): error sending drain request: rpc error: code = DeadlineExceeded desc = context deadline exceeded
Failed running "node drain"
```

I will leave as is.